docs/keyboard-shortcuts.md

```diff
@@ -0,0 +1,59 @@
+| `Ctrl+F` / `Right Arrow`                           | Move the cursor one character to the right.                                                                                         |
+| `Ctrl+Left Arrow` / `Meta+Left Arrow` / `Meta+B`   | Move the cursor one word to the left.                                                                                               |
+| `Ctrl+Right Arrow` / `Meta+Right Arrow` / `Meta+F` | Move the cursor one word to the right.                                                                                              |
+| `Ctrl+U`                                           | Delete from the cursor to the beginning of the line.                                                                                |
+| `Ctrl+K`                                           | Delete from the cursor to the end of the line.                                                                                      |
+| `Ctrl+W` / `Meta+Backspace` / `Ctrl+Backspace`     | Delete the word to the left of the cursor.                                                                                          |
+| `Meta+Delete` / `Ctrl+Delete`                      | Delete the word to the right of the cursor.                                                                                         |
+| `Ctrl+H` / `Backspace`                             | Delete the character to the left of the cursor.                                                                                     |
+| `Ctrl+D` / `Delete`                                | Delete the character to the right of the cursor.                                                                                    |
+| `Ctrl+X` / `Meta+Enter`                            | Open the current input in an external editor.                                                                                       |
+| `Ctrl+V`                                           | Paste clipboard content. If the clipboard contains an image, it will be saved and a reference to it will be inserted in the prompt. |
+| `!`                                                | Toggle shell mode when the input is empty.                                                                                          |
+
+## Suggestions
+
+| Shortcut        | Description                            |
+| --------------- | -------------------------------------- |
+| `Up Arrow`      | Navigate up through the suggestions.   |
+| `Down Arrow`    | Navigate down through the suggestions. |
+| `Tab` / `Enter` | Accept the selected suggestion.        |
+
+## Radio Button Select
+
+| Shortcut           | Description          |
+| ------------------ | -------------------- |
+| `Up Arrow` / `k`   | Move selection up.   |
+| `Down Arrow` / `j` | Move selection down. |
+| `Enter`            | Confirm selection.   |
```

Comment:
also mention that you can select radio buttons with the numbers
1 through 9
Also mention for radio buttons with numbers > 9 you can still select if you press the two numbers one after another in quick succession.